### ISSUE TITLE
Menu styling update

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/myaccount#221]
During checkin meeting this week both @ruthtillman and @mkutch reported that contrast was too light in general, but specifically for:

* the menu
* book titles and metadata

Should also double check against WCAG AA

**Comment by cdmo** [comment]
After scanning pages of the application, I only found one (surprising) WCAG AA violation: 

![Screenshot_2020-07-29 My Account Pennsylvania State University Libraries](https://user-images.githubusercontent.com/523381/88798186-83723b80-d172-11ea-8a52-7d45c503bf9e.png)

The white on blue buttons on the summary page are 3.97:1 and failing to pass WCAG AA compliance.

**Comment by cdmo** [comment]
@ruthtillman can you remind me what we wanted to do for the menu bar? Just change the colors to be higher contrast and make the menu links equal width? Is that about right?

**Comment by ruthtillman** [comment]
Improve contrast of font & background (maybe explore other ways to show active page?), spread it out to better width--equal or somesuch

**Comment by cdmo** [comment]
Here's one way we could have the menu work: https://psu.box.com/s/e8n0ft538hcmuh4mqepjyqow6qvdr7kj

**Comment by cdmo** [comment]
Also wondering if now is a good time to rework the labeling of this column to be "Position in the holds queue" as the header with a "1st" or "3rd" or "99th" as the value for the rows. Think it'd be easier to read that way and make more sense. ?

![Screen Shot 2020-07-29 at 5 05 36 PM](https://user-images.githubusercontent.com/523381/88853526-dae9c900-d1bd-11ea-973b-5d95d758ad90.png)

**Comment by cdmo** [comment]
Starting to just compile questions here I guess. Do we want the book titles to be links? I know that was one of the complaints/feedback items.

**Comment by ruthtillman** [comment]
1) Yes, retitling the column vs. repeating all the text makes sense!
2) yes, book titles as links. I think they were a little on the light side for links? Or maybe it was font weight/size. What's in the screenshot looks good, though, so maybe it's ok?

**Comment by ruthtillman** [comment]
cutting down on noise ++

**Comment by ruthtillman** [comment]
Looking good. If we have any further questions/suggestions after testing, that can be handled separately or this can be reopened.
